docs/configuration/authentication/jwt.md

```diff
@@ -11,12 +11,16 @@ SECRET = "SECRET"
 
 auth_backends = []
 
-jwt_authentication = JWTAuthentication(secret=SECRET, lifetime_seconds=3600)
+jwt_authentication = JWTAuthentication(secret=SECRET, lifetime_seconds=3600, tokenUrl="auth/jwt/login")
 
 auth_backends.append(jwt_authentication)
 ```
 
-As you can see, instantiation is quite simple. You just have to define a constant `SECRET` which is used to encode the token and the lifetime of token (in seconds).
+As you can see, instantiation is quite simple. You have to define three arguments:
```

Comment:
You don't have to define three arguments - `secret` and `lifetime_seconds` are the only required arguments.  `tokenUrl` and `name` are optional.  Perhaps it's also worth mentioning the arguments by their name as well?